script/Ciel/Day 13/13_01_02_CIEL13_7F.txt

```diff
@@ -175,7 +175,7 @@ Perhaps 'trauma' is the word I'm looking for here.
 [sha:6db6df99649bfe0f90f968c71d24ac356d499cd1]{
 -- Page 413, Offset 36480.
 -- ゴハンの前に着替えるのが先ですよねっ……！」
-I should get changed before we eat, shouldn't I!"
+I should get changed before we eat, shouldn't I?!"
```

Comment:
! can have implicit ?, using both here makes it read a lot more... confused? than I think it should come off as